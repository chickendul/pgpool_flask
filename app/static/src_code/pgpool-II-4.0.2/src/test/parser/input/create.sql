version https://git-lfs.github.com/spec/v1
oid sha256:f038311a4c22ecc3268588a8907c01ae14f54ced6e92ad05c3495304452fa408
size 2907
