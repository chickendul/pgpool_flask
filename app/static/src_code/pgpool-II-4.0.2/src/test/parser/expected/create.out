version https://git-lfs.github.com/spec/v1
oid sha256:78eb95ff7326b72d5b896f0869c3b124282a96cac192ab598a102394fa028eb4
size 3325
